docs/api/QUIC_STATELESS_RETRY_CONFIG.md

```diff
@@ -45,6 +45,20 @@ Changing `RotationMs`, `Algorithm`, or `Secret` will invalidate all retry tokens
 All servers deployed in a cluster and sharing the secret must have their clocks synchronized within `RotationMs` of UTC.
 A server whose clock is ahead of UTC may produce a retry token that other servers in that deployment are unable to validate.
 
+## Stateless Retry key Generation Algorithm
+
+The stateless retry key is generated from the above configuration parameters using the [SP800-108 rev. 1 CTR-HMAC KDF](https://csrc.nist.gov/pubs/sp/800/108/r1/upd1/final) algorithm with SHA256.
```

Comment:
It might be worth while to give explicit, binary examples and walk through the steps and provide output. This generally helps others validate they are producing exactly the same thing as you.

Reply:
Added a couple examples with concrete settings